test/cutting/cut_finding/test_quantum_device_constraints.py

```diff
@@ -0,0 +1,35 @@
+# This code is a Qiskit project.
+
+# (C) Copyright IBM 2024.
+
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
```

Comment:
We could move these tests into `test/test_find_cuts.py`

Reply:
Yes, on it.

Reply:
'Tis done: [002144e](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/520/commits/002144eab5a2b2a5a5304226ca6a084e21db8c7c)